docs/usage/applications.rst

```diff
@@ -41,39 +44,37 @@ order, once the ASGI server (uvicorn, hypercorn, etc.) emits the respective even
 A classic use case for this is database connectivity. Often, we want to establish a database connection on application
 startup, and then close it gracefully upon shutdown.
 
-For example, lets create a database connection using the async engine from
+For example, let us create a database connection using the async engine from
 `SQLAlchemy <https://docs.sqlalchemy.org/en/latest/orm/extensions/asyncio.html>`_. We create two functions, one to get or
-establish the connection, and another to close it, and then pass them to the Litestar constructor:
+establish the connection, and another to close it, and then pass them to the :class:`~litestar.app.Litestar` constructor:
 
 .. literalinclude:: /examples/startup_and_shutdown.py
```

Comment:
IDK why but this (and most of the other examples) on the page don't have full syntax highlighting in the docs preview. The Hello World example at the top does, but most others don't. E.g., keyword highlights, etc.

Reply:
Im not seeing why it does this either :|

Reply:
tried tagging it with `:language:`, matching what the working one does, nothing.. not sure if its a local thing?